src/managers/auth-manager.js

```diff
@@ -357,7 +320,39 @@ AuthManager.prototype = $.extend(AuthManager.prototype, {
      * @param {Object} `options` (Optional) Overrides for configuration options.
      */
     getCurrentUserSessionInfo: function (options) {
-        return getSession(this.store, options);
+        return this.sessionManager.getSession(options);
+    },
+
+
+    /**
+     * Will add a group to the session, it is assumed that the group and project exist and the user is part of it.
+     *
+     * Returns the new session object.
+     *
+     * **Example**
+     *
+     *      authMgr.addGroup('acme', { groupName: });
+     *
+     * **Parameters**
+     * @param {Object} `group` (Required) must have the project, groupName and groupId properties, isFac is optional and defaults to false
+     */
+    addGroup: function (group) {
+        return this.addGroups([group]);
+    },
+
+    addGroups: function (groups) {
```

Comment:
Why not have a single function which works with array/non-array similar to other services (like for instance `addUsers` in world-api-adapter)?